data/common.yaml

```diff
@@ -59,6 +59,7 @@ ntp::tos_floor: 1
 ntp::tos_maxclock: 6
 ntp::tos_minclock: 3
 ntp::tos_minsane: 1
+ntp::tos_orphan: 6
```

Comment:
Is having this enabled by default at a value of `6` for all users and OS's a reasonable default? This would affect all users as-is.

Reply:
I wouldn't. This option was added in ntpd 4.2.2, I think, and might be not available everywhere, so it can be a breaking change.

Reply:
ntpd 4.2.2 was released in October 2005. An older version shouldn't be around anymore. The default should have been fine for all users as you shouldn't have an upstream ntp with a stratum worse than 4 as source for your pool.

It would have clashed with users of the undisciplined local clock which shouldn't be used with 4.2.2 and later (ntpd documentation).